packages/flutter_tools/test/commands.shard/hermetic/drive_test.dart

```diff
@@ -459,6 +459,116 @@ void main() {
         ProcessManager: () => FakeProcessManager(<FakeCommand>[]),
       });
     });
+
+    group('debugging options', () {
+      const Symbol kDebugOptionsSymbol = Symbol('debuggingOptions');
+      DebuggingOptions debuggingOptions;
+
+      String testApp, testFile;
+
+      setUp(() {
+        restoreAppStarter();
+      });
+
+      Future<void> appStarterSetup() async {
+        mockDevice = MockDevice();
+        testDeviceManager.addDevice(mockDevice);
+
+        final MockDeviceLogReader mockDeviceLogReader = MockDeviceLogReader();
+        when(mockDevice.getLogReader()).thenReturn(mockDeviceLogReader);
+        final MockLaunchResult mockLaunchResult = MockLaunchResult();
+        when(mockLaunchResult.started).thenReturn(true);
+        when(mockDevice.startApp(
+          null,
+          mainPath: anyNamed('mainPath'),
+          route: anyNamed('route'),
+          debuggingOptions: anyNamed('debuggingOptions'),
+          platformArgs: anyNamed('platformArgs'),
+          prebuiltApplication: anyNamed('prebuiltApplication'),
+        )).thenAnswer((Invocation invocation) async {
+          debuggingOptions = invocation.namedArguments[kDebugOptionsSymbol];
```

Comment:
nit: use the literal form of Symbol
```suggestion
          debuggingOptions = invocation.namedArguments[#debuggingOptions];
```